frontend/packages/console-shared/src/types/resource.ts

```diff
@@ -1,6 +1,7 @@
 import { K8sResourceKind, PodKind, RouteKind, EventKind } from '@console/internal/module/k8s';
 import { DEPLOYMENT_STRATEGY } from '../constants';
 import { OverviewItemAlerts, PodControllerOverviewItem } from './pod';
+import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
```

Comment:
argh, this is a bad dependency for `console-shared`

These sections should be contributed via extensions and not hard coded.

Reply:
@christianvogt As discussed, logged a bug for this https://issues.redhat.com/browse/ODC-3075.